src/status_im/qr_scanner/core.cljs

```diff
@@ -4,10 +4,11 @@
             [status-im.utils.utils :as utils]))
 
 (defn scan-qr-code
-  [identifier handler {:keys [db]}]
-  {:db                     (assoc-in db [:qr-codes identifier] handler)
+  [{:keys [modal?] :as identifier} m {:keys [db]}]
```

Comment:
It still doesn't tell us what m is

Reply:
I'm with @yenda, one letter symbols are very hard to read and even against official clojure code guidelines.